ui/package.json

```diff
@@ -29,12 +29,13 @@
     "not op_mini all"
   ],
   "devDependencies": {
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
Just a quick check - is this needed?

Reply:
@damienomurchu good question, I added it as a dependency based on [this github issue](https://github.com/facebook/create-react-app/issues/5259) as I was seeing the error described a while back. I did a smoke test by removing it and doing a fresh npm install and I'm not seeing the error now when running the tests.  We could remove it and if the issue arrises we could add it back in. It's a fix on an npm issue and not a dependency we're using in the project. It's also possible that a change in the code since seeing the issue removed the cause of the issue but I don't recall at exactly what stage I saw it.

Reply:
Yeah scratch that. I'm still seeing the error, so we need this dependency. 😂

Reply:
Hi @laurafitzgerald if this is an issue with NPM then it's an issue with your environment and not the project itself. You should update your own NPM version and see if that works using `@babel/core`.

What version of NPM are you on? I am on version 6.8.0.

What is the error you get? I can't reproduce.